src/main/webapp/examples/team.html

```diff
@@ -0,0 +1,146 @@
+        <div class="container text-center">
+            <div class="mb-5 text-left h2">
+                <span>Lorem ipsum dolor sit amet</span>
+            </div>
+
+            <div id="teamContent"></div>
+
+            <button id="btn" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
+        </div>
+    </section>
+</main>
+
+<footer class="footer">
+    <!--footer goes here -->
+</footer>
+
+<!-- to use the dynamic footer, use this instead of above tag => *<div id="footer"></div>* -->
+
+<!-- Core -->
+<script src="../assets/vendor/jquery/jquery.min.js"></script>
+<script src="../assets/vendor/popper/popper.min.js"></script>
+<script src="../assets/vendor/bootstrap/bootstrap.min.js"></script>
+<script src="../assets/vendor/headroom/headroom.min.js"></script>
+<!-- Argon JS -->
+<script src="../assets/js/argon.js?v=1.0.1"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+
+<!--Mustache template for team content-->
+<script type="text/html" id="templateTeam">
+    <div class="row">
+        {{#data}}
+        <div class="col-md-3 col-lg-3 mb-5">
+            <div class="px-4">
+                <div class="imgContainer">
+                    <img src="{{image}}"
+                         class="rounded-circle img-center img-fluid shadow shadow-lg--hover imgHover"
+                    >
+                    <div class="imgIcon">
+                        <a href="{{linkedin}}" class="btn btn-default btn-icon-only rounded-circle">
+                            <i class="fa fa-linkedin"></i>
+                        </a>
+                    </div>
+                </div>
+                <div class="pt-4 text-center">
+                    <h5 class="title">
+                        <span class="d-block mb-1">{{name}}</span>
+                        <small class="h6 text-muted">{{city}}</small>
+                    </h5>
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+
+<!--Script for load team data-->
+<script>
+    $.ajax({
+        type: 'get',
+        url: '../assets/content/json/dummy-profiles.json',
+        dataType: 'json',
+        success: function (data) {
+
+            //slice array to two parts
+            if(data.length>=8){
+                let partOne = data.slice(0, 8);
+                let partTwo = data.slice(8);
+
+                //mustache render
+                let contentPartOne = Mustache.render($("#templateTeam").html(), {"data": partOne});
+                let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
+
+                //display first 8 profiles
+                $("#teamContent").html(contentPartOne);
+
+                //hide button
+                $("#btn").click(function () {
+                    $("#teamContent").append(contentPartTwo);
+                    document.getElementById("btn").style.display = "none";
```

Comment:
You can use jQuery selector here: $("#btn").style.display = none;

Reply:
🤩